client/src/general_utils.ts

```diff
@@ -39,7 +39,7 @@ export const text_abbrev = function (text: string, length: number) {
 };
 
 export const make_unique_func = function () {
-  var val = 0;
+  let val = 0;
```

Comment:
ha, didn't know eslint prefers let over var

Reply:
If you see a `var` you can't interpret it as a `let`, have to assume it's legacy code (pre-`let`/`const`) and might actually be either a constant _or_ a mutable.

Reply:
Oh, but if you were wondering why it hadn't previously been applied, we disabled the rule in JS files because we wanted `var`s to be a sign that you were looking at legacy code that hadn't been refactored in years. I let the no-`var` rule apply to TS modules because we shouldn't be trying to type legacy code anyway, should be refactoring it first/in the process of typing it.

TS can give better type information on `const`s than on `let`s or `var`s too, I believe.